GoogleDataTransport/GDTCORLibrary/Public/GDTCORStorageProtocol.h

```diff
@@ -36,6 +38,31 @@ NS_ASSUME_NONNULL_BEGIN
 /** Removes the events from storage. */
 - (void)removeEvents:(NSSet<NSNumber *> *)eventIDs;
 
+/** Persists the given data with the given key.
```

Comment:
The new methods look unrelated to the previously existing ones. Do you think a separate protocol (e.g. `GDTCORKeyValueStorage`) and implementation may be a better abstraction from this perspective?

Reply:
That's not a bad idea--but these are methods that I want to require of all future storage implementations.